exporter/stackdriverexporter/observability.go

```diff
@@ -45,7 +38,22 @@ var viewPointCount = &view.View{
 	TagKeys:     []tag.Key{statusKey},
 }
 
-func recordPointCount(ctx context.Context, points int, status string) {
+func recordPointCount(ctx context.Context, success, dropped int, grpcErr error) {
+	if success > 0 {
+		recordPointCountDataPoint(ctx, success, "OK")
+	}
+
+	if dropped > 0 {
+		var st string
```

Comment:
I suppose you may initialise it with `st := "NONE"` or something like that, to make it explicit rather than empty string when error is missing.

Reply:
i agree that would be better but will leave this as is to keep consistent with the OC gRPC plugin as per above comment